website/docs/result_to_dict.md

```diff
@@ -0,0 +1,72 @@
+---
+id: result_to_dict
+title: Result to Dictionary
+sidebar_label: Result to Dictionary
+---
+## ResultToDict Decorator

```

Comment:
Before this add documentation to the function `result_to_dict`. only after that introduce the decorator 

+ provide simple example 
+ provide more complex example